pyro/contrib/autoname/__init__.py

```diff
@@ -0,0 +1,12 @@
+"""
+The :mod:`pyro.contrib.autoname` module provides tools for automatically
+generating unique, semantically meaningful names for sample sites.
+"""
+from __future__ import absolute_import, division, print_function
+
+from pyro.contrib.autoname import named
```

Comment:
This is fine, but I believe you can omit `named` from this file entirely and `from pyro.contrib.autoname import named` will still work.